specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2021-08-01/apimschema.json

```diff
@@ -0,0 +1,328 @@
+            }
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "Schema"
+        ],
+        "operationId": "GlobalSchema_CreateOrUpdate",
+        "description": "Creates new or updates existing specified Schema of the API Management service instance.",
+        "x-ms-examples": {
+          "ApiManagementCreateSchema1": {
+            "$ref": "./examples/ApiManagementCreateGlobalSchema1.json"
+          },
+          "ApiManagementCreateSchema2": {
+            "$ref": "./examples/ApiManagementCreateGlobalSchema2.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "./apimanagement.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ServiceNameParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/SchemaIdParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "./definitions.json#/definitions/GlobalSchemaContract"
+            },
+            "description": "Create or update parameters."
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/IfMatchOptionalParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "The new Schema was successfully added.",
+            "headers": {
+              "ETag": {
+                "description": "Current entity state version. Should be treated as opaque and used to make conditional HTTP requests.",
+                "type": "string"
+              }
+            },
+            "schema": {
+              "$ref": "./definitions.json#/definitions/GlobalSchemaContract"
+            }
+          },
+          "200": {
+            "description": "The Schema details were successfully updated.",
+            "headers": {
+              "ETag": {
+                "description": "Current entity state version. Should be treated as opaque and used to make conditional HTTP requests.",
+                "type": "string"
+              }
+            },
+            "schema": {
+              "$ref": "./definitions.json#/definitions/GlobalSchemaContract"
+            }
+          },
+          "202": {
+            "description": "Accepted the change when updating the service."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./apimanagement.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
```

Comment:
>"x-ms-long-running-operation": [](http://example.com/codeflow?start=8&length=30)

Can you set x-ms-long-running-operation-options to location? By default it is azure-async-operation.

[https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-long-running-operation-options](https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-long-running-operation-options)

Reply:
Done